content/news/0.10.0_release.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "Release 0.10.0"
+date = 2019-03-06
+publishdate = 2018-03-06
+author = "@hsanjuan"
++++
+
+## 20190306 | Release 0.10.0
+
+Today we release 0.10.0, a release with major changes under the hood that will
+make IPFS Cluster perform significantly faster with large pinsets and less
+memory demanding.
+
+For those upgrading, this release is a mandatory step before any future
+upgrades, as it will upgrade the internal state to a new format which prepares
+the floor for the upcoming addition of an alternative CRDT-based "consensus"
+component. The new component will increase IPFS Cluster scalability orders of
+magnitude and unlock collaborative Cluster where random invididuals can
+collaborate in replicating content.
```

Comment:
Should we be writing what we are planning to do in release notes? or just what we did?

Reply:
These aren't 'release' notes per se but a news article/blog post announcing the release